From b093c43effdb30c363165ae8f37c206339360c97 Mon Sep 17 00:00:00 2001
From: stic-server-open <1138705738@qq.com>
Date: Sat, 29 Apr 2023 14:06:56 +0800
Subject: [PATCH 115/157] arch: arm64: configs: xiaomi: disable per-cgroup
 pressure tracking

* Filled in the missing cmdline due to overwriting config:
  0a19015b ("defconfig: disable per-cgroup pressure tracking")

Author: Srinivasarao P <spathi@codeaurora.org>:
* defconfig: disable per-cgroup pressure tracking

  Disable per-cgroup psi tracking to prevent unnecessary overhead since
  Android does not use per-cgroup psi information.

Author: 2869145032 <2869145032@qq.com>
* arch: arm64: configs: xiaomi: Remove duplicate CONFIG_CMDLINE_EXTEND

  kona-perf_defconfig has set CONFIG_CMDLINE_EXTEND=y

- check:
  Successful boot, check /proc/config.gz -> config:
   former: CONFIG_CMDLINE="ramoops_memreserve=4M"
   later: CONFIG_CMDLINE="cgroup_disable=pressure ramoops_memreserve=4M"

  boot cmdline: cgroup_disable=pressure ramoops_memreserve=4M ...
  kernel dmesg: Disabling pressure control group feature

Co-Authored-By: Srinivasarao P <spathi@codeaurora.org>
Co-Authored-By: 2869145032 <2869145032@qq.com>
Change-Id: Ic126e3cb7f5cd32dd4fab396ae78f00d673a337e
---
 arch/arm64/configs/vendor/xiaomi/sm8250-common.config | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/configs/vendor/xiaomi/sm8250-common.config b/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
index 77349610734e..bb8795ef5f8e 100644
--- a/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
+++ b/arch/arm64/configs/vendor/xiaomi/sm8250-common.config
@@ -26,8 +26,7 @@ CONFIG_PSTORE=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_PMSG=y
 CONFIG_PSTORE_RAM=y
-CONFIG_CMDLINE="ramoops_memreserve=4M"
-CONFIG_CMDLINE_EXTEND=y
+CONFIG_CMDLINE="cgroup_disable=pressure ramoops_memreserve=4M"
 CONFIG_BLK_WBT=y
 CONFIG_BLK_WBT_SQ=y
 CONFIG_MTD=y
-- 
2.48.1

